string spiDevice
uint16 status_register
uint16 delta_x_disp_register
uint16 delta_y_disp_register
uint16 command_high_register
uint16 command_low_register
uint16 memory_pointer_register
uint16 memory_data_register
uint16 mode_control_register
uint16 power_control_register
uint16 mode_status_register
uint16 system_control_register
uint16 miscellaneous_register
uint16 interrupt_output_register
